vertx-web-client/src/test/java/io/vertx/webclient/RxTest.java

```diff
@@ -83,4 +83,36 @@ public void testPost() {
       server.close();
     }
   }
+
+  @Test
+  public void testResponseMissingBody() throws Exception {
+    int times = 5;
+    waitFor(times);
+    HttpServer server = vertx.createHttpServer(new HttpServerOptions().setPort(8080));
+    server.requestStream().handler(req -> {
+      req.response().setStatusCode(403).end();
+    });
+    try {
+      server.listen(ar -> {
+        client = WebClient.wrap(vertx.createHttpClient(new HttpClientOptions()));
+        Single<HttpResponse<Buffer>> single = client
+          .get(8080, "localhost", "/the_uri")
+          .rxSend();
+        for (int i = 0; i < times; i++) {
+          single.subscribe(resp -> {
+            assertEquals(403, resp.statusCode());
+            try {
+              assertNull(resp.body());
+            } catch (Exception e) {
```

Comment:
good catch, it's because I used it for debugging purpose when fixing this bug https://github.com/vert-x3/vertx-rx/commit/de0d5ce79c01d45370c7ac51ba3b8dbb8c19f788